biz.aQute.bndlib/src/aQute/bnd/osgi/Analyzer.java

```diff
@@ -2562,6 +2562,9 @@ public String _packages(String... args) throws Exception {
 							throw new IllegalArgumentException("Not enough arguments in ${packages} macro");
 						accept = pkgInfo != null && pkgInfo.is(Clazz.QUERY.ANNOTATED, instr, this);
 						break;
+					case EXPORTED :
+						accept = entry.getValue().getVersion() != null;
```

Comment:
It could, but it's very unlikely. Got this hint from @pkriens, which probably means there's not currently a better way to know if a package is exported.